strapi/strapi
# fix: blocks fields not working in GraphQL

### What does it do?

Adds blocks to the list of known strapi types in the GraphQL plugin. And point it to the JSON GraphQL scalar.

### Why is it needed?

Blocks fields were not working in GraphQL

### How to test it?

- Run the getstarted example app
- Have a published kitchensink entry with content inside the blocks attribute
- Open the [GraphQL playground](http://localhost:1337/graphql)
- Write this query:

```graphql
query {
  kitchensinks {
    data {
      attributes {
        blocks
      }
    }
  }
}
```

When running it you should see your blocks data in the response.

### Related issue(s)/PR(s)

fixes #18430
